apps/pharos_web/web/core/real_search.ex

```diff
@@ -3,19 +3,40 @@ defmodule Core.RealSearch do
   def execute(%{"topic" => topic, "amount" => amount}) do
     query = %{topic: topic, amount: String.to_integer(amount)}
 
```

Comment:
Maybe it's better to keep this code on our side only - and we can use the slides to explain each step and get them to implement the parts rather than uncomment them. Let me know what you think and we can update this PR accordingly.